File: net/src/Sdl.Web.PublicContentApi/ContentModel/IContextData.cs
using System.Collections.Generic;

namespace Sdl.Web.PublicContentApi.ContentModel
{
    /// <summary>
    /// Context Data
    /// </summary>
    public interface IContextData
    {
        /// <summary>
        /// List of claim values to pass to query.
        /// </summary>
        List<ClaimValue> ClaimValues { get; set; }
    }
}

File: java/pca-model/src/main/java/com/sdl/web/pca/client/contentmodel/IContextData.java
package com.sdl.web.pca.client.contentmodel;


import java.util.List;

/// Context Data
public interface IContextData {
    /// List of claim values to pass to query.
     List<ClaimValue> getClaimValues();

     void setClaimValues(List<ClaimValue> claimValues);
}
